src/test/suite/fixtures/recorded/languages/rust/chuckFunkNameFine.yml

```diff
@@ -0,0 +1,33 @@
+languageId: rust
```

Comment:
Nice.  Worth also testing that you can just use "name" for this one as well

Also, not worth re-recording, but we prefer "change" to "chuck" if you're not explicitly trying to test removal semantics, as otherwise you're implicitly testing our whitespace removal code